Add browser SDK setup options to Cdocs browser partial (#37270)
DataDog/documentation: layouts/shortcodes/mdoc/en/sdk/setup/browser.mdoc.md

Author: ```suggestion
```

It can be included directly in language-specific pages or wrapped in conditionals.
-->

> This page describes how to instrument your web applications with the Datadog Browser SDK. The Browser SDK supports [Real User Monitoring (RUM)][1], [Error Tracking][2], [Session Replay][3], and [Product Analytics][4].

Reviewer: Just confirming that [1] linking to `/real_user_monitoring/application_monitoring/browser/setup/client` is desired? As opposed to the RUM landing page

Author: thanks for the catch!

Select a setup method based on your application stack and workflow:

- **Manual client-side setup**: Add the SDK directly to your frontend code when you need full control over initialization and configuration.
- **Agentic Onboarding**: Use an AI-assisted workflow to automatically configure and deploy the SDK with minimal manual steps.

> - **Server-side auto-instrumentation (RUM only)**: Inject the SDK through your web server or proxy when you can't modify frontend code directly.

Reviewer: ```suggestion
- **Server-side auto-instrumentation (RUM only)**: Inject the SDK through your web server or proxy when you cannot modify frontend code directly.
```

#### Configure tracking consent (GDPR compliance)

To be compliant with GDPR, CCPA, and similar regulations, the Browser SDK lets you provide the [tracking consent value at initialization][8].

#### Configure Content Security Policy (CSP)

If you're using the Datadog Content Security Policy (CSP) integration on your site, see [the CSP documentation][9] for additional setup steps.
{% /step %}

{% /stepper %}

{% /collapse-content %}

{% collapse-content title="Option 2: Agentic Onboarding" level="h3" %}

Use the [Agentic Onboarding][10] page to instrument your browser application using the AI Setup CLI or the Datadog MCP Server.

{% /collapse-content %}

{% collapse-content title="Option 3: Server-side auto-instrumentation" level="h3" %}

{% site-region region="gov,gov2" %}
{% alert level="danger" %}
RUM Auto-Instrumentation is not available for the selected site ({% region-param key="dd_site_name" /%}). Use [Client-Side instrumentation][1] instead.
{% /alert %}
{% /site-region %}

{% alert level="info" %}

> Server-side auto-instrumentation only supports RUM. For Error Tracking, Session Replay, or Product Analytics, use Option 1 (manual client-side setup).

Reviewer: Referencing "Option 1" by number feels... fragile. Maybe just `use [manual client-side setup][1]` suffices?

> Server-side auto-instrumentation only supports RUM. For Error Tracking, Session Replay, or Product Analytics, use Option 1 (manual client-side setup).

Author: ```suggestion
Server-side auto-instrumentation only supports RUM. For Error Tracking, Session Replay, or Product Analytics, use [manual client-side setup][14].
```

> [13]: /real_user_monitoring/application_monitoring/agentic_onboarding/?tab=realusermonitoring

Reviewer 2: ```suggestion
```
The body no longer references links 12 and 13, so you should remove them.
